From 0ff428bd64a5cff6ffde1e53c39d4cf1187ea3e9 Mon Sep 17 00:00:00 2001
From: EvoX12 Spike <spike@localhost>
Date: Mon, 24 Aug 2026 21:49:39 +0300
Subject: [PATCH] qcom_defs: add canoe to UM_6_6_FAMILY (carried from the A16
 tree local edit)

Change-Id: Idad29ae6890df96268b7ca483c4dc05b845c7321
(cherry picked from commit d5cb645503ed14dd5a8e625b3a27090b1c1d5883)
---
 qcom_defs.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qcom_defs.mk b/qcom_defs.mk
index 66d7dbd..4416399 100644
--- a/qcom_defs.mk
+++ b/qcom_defs.mk
@@ -12,7 +12,7 @@ UM_5_4_FAMILY := lahaina holi
 UM_5_10_FAMILY := taro parrot
 UM_5_15_FAMILY := kalama crow
 UM_6_1_FAMILY := pineapple volcano
-UM_6_6_FAMILY := sun
+UM_6_6_FAMILY := sun canoe
 
 ifeq ($(TARGET_KERNEL_VERSION),4.19)
 # UM 4.9 upgraded to UM 4.19
-- 
2.55.0

